velox/functions/sparksql/RegisterArithmetic.cpp

```diff
@@ -110,6 +109,13 @@ void registerArithmeticFunctions(const std::string& prefix) {
   VELOX_REGISTER_VECTOR_FUNCTION(udf_decimal_div, prefix + "divide");
   registerFunction<sparksql::IsNanFunction, bool, float>({prefix + "isnan"});
   registerFunction<sparksql::IsNanFunction, bool, double>({prefix + "isnan"});
+
+  // Register checked arithmetic functions for Spark arithmetic exprs with try
+  // eval mode.
```

Comment:
nit: since this comment is about how we use the registered functions in Gluten, perhaps we can remove it.